sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -262,7 +262,7 @@ describeFuncName
     ;
 
 describeColName
-    : identifier ('.' (identifier | STRING))*
+    : identifier
```

Comment:
That was my previous approach. As @cloud-fan mentioned in [#discussion_r122347812](https://github.com/apache/spark/pull/16422#discussion_r122347812) and [#discussion_r122348197](https://github.com/apache/spark/pull/16422#discussion_r122348197), it's hard to tell if "col.a" is a qualified name or a nested name, or maybe a more complex example:
```
`a.b`.`c.d`
```
```
`a.b.c.d`
```
It may be very tricky to deal with such cases. What's your opinion?